### PR TITLE
Improvements for AAB bundle process

### PR DESCRIPTION
**Pull request by AGulev**
- Sign AAB file
- Copy debug symbols into bundle folder
- Cleanup bundle folder from intermediate artefacts

---

According to https://developer.android.com/studio/command-line/bundletool AAB should be signed as well:

>  you can also build your app bundle from the command line and sign it using jarsigner

We use our own tool instead  `jarsigner`. There is no information about aligning. After googling I found that we shouldn't align AAB.

### CODE REVIEW

File: com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidAAB.java [hunk 1]
```diff
@@ -433,5 +473,14 @@ public static void create(Project project, File bundleDir, ICanceled canceled) t
 
 		// STEP 5. Use bundletool to create AAB from base.zip
 		File baseAab = createBundle(project, appDir, baseZip, canceled);
+
+		// STEP 6. Copy debug symbols
+		final boolean has_symbols = project.hasOption("with-symbols");
+		if (has_symbols) {
+			copySymbols(project, appDir, title);
+		}
```
Comment (AGulev) [inline com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidAAB.java:481]: We take into account `has_symbols` flag and copy-paste debug symbols into the bundle folder.

File: com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidAAB.java [hunk 2]
```diff
@@ -433,5 +473,14 @@ public static void create(Project project, File bundleDir, ICanceled canceled) t
...
+
+		// STEP 7. Cleanup bundle folder from intermediate folders and artifacts.
+		cleanupBundleFolder(androidResDir, appDir);
```
Comment (AGulev) [inline com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidAAB.java:484]: We remove intermediate artifact folders `res` and `aab` from our bundle folder

File: com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidAPK.java [hunk 3]
```diff
@@ -357,6 +357,8 @@ public static void create(Project project, File bundleDir, ICanceled canceled) t
         ap3.delete();
         ap4.delete();
         FileUtils.deleteDirectory(tmpResourceDir);
+        FileUtils.deleteDirectory(new File(appDir, "libs"));
+        FileUtils.deleteDirectory(resDir);
```
Comment (AGulev) [inline com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/AndroidAPK.java:361]: We also remove intermediate artefacts folders `res` and `libs` from our bundle folder for apk build

**Review by britzl — APPROVED** [review]
Added a fix for paths with spaces. Otherwise a great PR!

**Pull request merged**